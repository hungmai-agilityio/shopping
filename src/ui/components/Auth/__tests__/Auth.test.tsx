import AuthForm from '@/ui/components/Auth';
import { render } from '@testing-library/react';

describe('Auth component', () => {
  const defaultProps = {
    title: 'Form',
    subtitle: 'Sub form',
    backgroundImage: '/product.webp',
    children: 'form login'
  };
  test('should be render match to snapshot', () => {
    const { container } = render(<AuthForm {...defaultProps} />);

    expect(container).toMatchSnapshot();
  });
});
