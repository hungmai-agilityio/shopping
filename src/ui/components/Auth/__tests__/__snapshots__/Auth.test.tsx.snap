// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Auth component should be render match to snapshot 1`] = `
<div>
  <section
    class="className bg-no-repeat bg-cover text-white"
    style="background-image: url(/product.webp);"
  >
    <div
      class="bg-gradient-overlay py-20"
    >
      <div
        class="w-full container"
      >
        <div
          class="md:max-w-card-sm"
        >
          <nav
            aria-label="breadcrumb"
            class="flex items-center space-x-2 text-white"
          >
            <span
              class="flex items-center"
            >
              <a
                class="hover:underline"
                href="/"
              >
                Home
              </a>
              <span
                class="mx-2 text-white"
              >
                /
              </span>
            </span>
            <span
              class="flex items-center"
            >
              <span
                class="font-bold text-white"
              >
                My Account
              </span>
            </span>
          </nav>
          <h1
            class="font-bold text-white text-2xl className capitalize lg:text-heading text-xl my-5"
          >
            My Account
          </h1>
          <h2
            class="font-bold text-white text-4xl className mt-11"
          >
            Form
          </h2>
          <div
            class="mt-6"
          >
            Sub form
          </div>
          <div
            class="mt-10 mb-20"
          >
            form login
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
